api/nodes/serializers.py

```diff
@@ -212,6 +206,7 @@ def profile_image_url(self, user):
 
     class Meta:
         type_ = 'contributors'
+        target_type_ = 'users'
```

Comment:
Hypothetical question: what would happen if I wanted to have two `JSONAPIHyperlinkedIdentityField` fields in a single serializer, each with a different `target_type_`?

By making the field depend on a value in `Meta`, it seems that we are artificially restricting serializers to one type of relationship, with some of the field behavior governed by a setting that is not part of the field. Can you provide insight into this design choice?

(why not just make target_type an argument to the field constructor?)